samples/3_map_state_simple/package.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
Just a few thoughts on including the build bits (package.json, karma, protractor, etc). Those files can be a pain to maintain and refresh. 

Option A:
- Leave as is

Option B:
- Just include the source files and let people copy the files into their own projects

Option C:
- Create a tabbed application that runs each sample in its own tab.

Option D:
- ??